packages/browser-sdk/src/client.ts

```diff
@@ -423,20 +451,24 @@ export class ReflagClient {
 
     this.flagsClient = new FlagsClient(
       this.httpClient,
-      // API expects `other` and we have `otherContext`.
-      {
-        user: this.context.user,
-        company: this.context.company,
-        other: this.context.otherContext,
-      },
+      this.context,
       this.logger,
-      {
-        expireTimeMs: opts.expireTimeMs,
-        staleTimeMs: opts.staleTimeMs,
-        fallbackFlags: opts.fallbackFlags,
-        timeoutMs: opts.timeoutMs,
-        offline: this.config.offline,
-      },
+      isBootstrapped(opts)
+        ? {
+            bootstrappedFlags: opts.bootstrappedFlags,
+            bootstrappedOverrides: opts.bootstrappedOverrides,
+            offline: this.config.offline,
+            overridesProvider: opts.overridesProvider,
+          }
+        : {
+            expireTimeMs: opts.expireTimeMs,
+            staleTimeMs: opts.staleTimeMs,
+            staleWhileRevalidate: opts.staleWhileRevalidate,
+            timeoutMs: opts.timeoutMs,
+            fallbackFlags: opts.fallbackFlags,
+            offline: this.config.offline,
+            overridesProvider: opts.overridesProvider,
+          },
```

Comment:
this could probable be simplified if the `opts` was just `InitOptions` with optional properties. 

Generally, I dislike these union types for configs because the relationship quickly becomes complicated. For example, should we also have an `InitOptionsOffline` which only contains the stuff needed in case it's running in `offline` mode etc.

Reply:
Good point. I don't have strong preferences either way. Thoughts at @pavkam. I could see a benefit to having the React props differentiate a bit between the two providers.